### ISSUE TITLE
Will there be any support for django 2.2 and above?

### ISSUE THREAD

**Issue by apowell656** [django-recurrence/django-recurrence#142]
Do you plan on any updates so that current versions of django will be supported? I have been trying and hoping to get this to work with 2.2 and just re-read the documentation to find this...

```
Currently, django-recurrence supports Python 2.7, Python 3.3, Python 3.4 and Python 3.5.

django-recurrence works with Django from versions 1.7 to 1.11.
```

It appears that {{form.media}} is loading enough to create the textarea but that is it.

![Screenshot from 2019-06-18 18-30-28](https://user-images.githubusercontent.com/1969515/59724263-41e53200-91f7-11e9-90a3-915abf40f45f.png)

![Screenshot from 2019-06-18 18-31-56](https://user-images.githubusercontent.com/1969515/59724314-5f1a0080-91f7-11e9-9b50-d78cef80069c.png)

**Comment by dominicrodger** [comment]
I think Django up to at least 2.1 is supported, and I suspect 2.2 works too - that documentation is just out of date. If you install an older version of Django locally, does that page work? The error you're seeing doesn't look Django version related to me - I suspect you've just got some set up issues.

**Comment by apowell656** [comment]
I followed the instructions  [here](https://django-recurrence.readthedocs.io/en/latest/installation.html) and added the following to my project urls.py (not my applications urls.py)

`import django
from django.conf.urls import url
from django.views.i18n import JavaScriptCatalog

# Your normal URLs here...

# If you already have a js_info_dict dictionary, just add
# 'recurrence' to the existing 'packages' tuple.
js_info_dict = {
    'packages': ('recurrence', ),
}

# jsi18n can be anything you like here
urlpatterns += [
    url(r'^jsi18n/$', JavaScriptCatalog.as_view(), js_info_dict),
]`
Everything else in my version is working fine. I could try installing an older version later, but I am hoping there is something that I am missing.

**Comment by apowell656** [comment]
The [link](https://github.com/django-recurrence/django-recurrence/issues/47) helped me get past the pgettext error in the console now I just have 

```
Uncaught ReferenceError: django is not defined
    at <anonymous>:1:1
    at p (jquery.min.js:2)
    at Function.globalEval (jquery.min.js:2)
    at text script (jquery.min.js:4)
    at Nb (jquery.min.js:4)
    at A (jquery.min.js:4)
    at XMLHttpRequest.<anonymous> (jquery.min.js:4)
    at Object.send (jquery.min.js:4)
    at Function.ajax (jquery.min.js:4)
    at Function.r._evalUrl (jquery.min.js:4)
```
Looking through the js files I am unable to tell what jquery dependency is needed. It feels that maybe more or less the problem. I am currently using jQuery 3.4.1 .

**Comment by apowell656** [comment]
Ok, so I removed all occurrences of django.jQuery and added scripts/CSS to my static/lib folders and things are displaying. I will just have to work with the CSS which is not playing nice with the modal. I am guessing there needs to be a check if the form is in the admin or not for the widget to load properly.

**Comment by apowell656** [comment]
@dominicrodger I have been able to get everything working except for the calendar button ( I am not a javascript guy) any ideas or help to get this piece working?

**Comment by aaugustin** [comment]
I'm using Python 3.6 and Django 2.2. I hit the same issue. Pull request #143 fixes it.

**Comment by dominicrodger** [comment]
Awesome - thanks @aaugustin. Once I've merged #143 I'll update the documentation to make it clearer which versions of Django are supported.

**Comment by dominicrodger** [comment]
Merged #143, docs updated in 07add25c01058206d3e273598c8716d8d31c47e8.

**Comment by apowell656** [comment]
So the package should be able to be executed as designed without my
"hackery"?

AP
